pkg/platforms/kubernetes/client_alphav2_linux.go

```diff
@@ -165,3 +173,25 @@ func getNS(origctx context.Context, cli criapi.RuntimeService, ns string, contai
 	inod, err := strconv.ParseInt(matches[1], 10, 0)
 	return uint64(inod), err
 }
+
+func getEnv(cli criapi.RuntimeService, containerid string) (map[string]string, error) {
```

Comment:
That was my first thought as well,  but I preferred this since its final source of truth. We can have those options set somewhere in docker image too and that wouldn't be picked up.

Reply:
Since env vars can contain secrets I would prefer not to have squash running printenv without good reason and proper documentation.

Reply:
Anyways we are attaching a debugger here, I can always get all env variables in debugger. I doubt there is any security enforcement actually possible. I can simply call os.getEnviron() in debug shell.

Reply:
are we good with conversation, can we mark it resolved? Any todo here?